db/schema.rb

```diff
@@ -171,8 +173,8 @@
     t.integer "person_id"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.index ["finding_aid_id"], name: "index_finding_aid_responsibilities_on_finding_aid_id"
-    t.index ["person_id"], name: "index_finding_aid_responsibilities_on_person_id"
+    t.index ["finding_aid_id"], name: "index_finding_aid_responsibility_on_finding_aid_id"
```

Comment:
Why did this change?

Reply:
It does not look there has been a change. The lines removed are added back just as they were, and in the same place. I.e. I don't know, but it shouldn't have any effect.